### PR TITLE
Save labels and other related documents on tracking number

### PR DESCRIPTION
**Pull request by bhavana94**

### CODE REVIEW

File: stock.py [hunk 1]
```diff
@@ -291,7 +291,9 @@ def generate_shipping_labels(self, **kwargs):
                 Attachment.create([{
                     'name': "%s_%s_USPS-Endicia.png" % (tracking_number, id),
                     'data': buffer(base64.decodestring(label)),
-                    'resource': '%s,%s' % (self.__name__, self.id)
+                    'resource': '%s,%s' % (self.tracking_number.__name__,
```
Comment (uzayr) [inline stock.py]: this too!!

**Comment by riteshshrv** [comment]
LGTM

File: stock.py [hunk 2]
```diff
@@ -291,7 +291,9 @@ def generate_shipping_labels(self, **kwargs):
                 Attachment.create([{
                     'name': "%s_%s_USPS-Endicia.png" % (tracking_number, id),
                     'data': buffer(base64.decodestring(label)),
-                    'resource': '%s,%s' % (self.__name__, self.id)
+                    'resource': '%s,%d' % (self.tracking_number.__name__,
+                    self.tracking_number.id
```
Comment (tarunbhardwaj) [inline stock.py]: Fix indentation, code should be beautiful

**Comment by tarunbhardwaj** [comment]
merged manually

**Pull request closed without merging**